src/app/shared/components/template/components/button/button.component.ts

```diff
@@ -37,6 +37,8 @@ interface IButtonParams {
   iconSecondary: string;
   /** TEMPLATE PARAMETER: "image_asset". The path to an image asset */
   image: string;
+  /** TEMPLATE PARAMETER: "button_icon_align". ALigns the icon to the left or right */
+  buttonIconAlign: "left" | "right";
```

Comment:
nit(blocking): Should be renamed to `iconAlign`/`icon_align`, since we're already namespaced inside the button component